.ci/docker/common/install_xpu.sh

```diff
@@ -41,13 +41,16 @@ function install_ubuntu() {
         libegl-mesa0 libegl1-mesa libegl1-mesa-dev libgbm1 libgl1-mesa-dev libgl1-mesa-dri \
         libglapi-mesa libgles2-mesa-dev libglx-mesa0 libigdgmm12 libxatracker2 mesa-va-drivers \
         mesa-vdpau-drivers mesa-vulkan-drivers va-driver-all vainfo hwinfo clinfo
+    if [[ "${XPU_DRIVER_TYPE,,}" == "rolling" ]]; then
+        apt-get install -y intel-ocloc
+    fi
```

Comment:
Because the installation between LTS and latest rolling driver on ubuntu are different now. Previously, the ocloc package included in igc package on ubuntu both for LTS and rolling.  But in the latest ubuntu rolling driver, the ocloc has been extracted as a standalone package. For others 2 OSes, the ocloc is always a standalone package.